src/Slices/ServiceInventory/UI/Components/DeleteModal/DeleteModal.tsx

```diff
@@ -13,11 +13,13 @@ import { GetInstancesContext } from "@S/ServiceInventory/UI/GetInstancesContext"
 
 interface Props extends VersionedServiceInstanceIdentifier {
   isDisabled?: boolean;
+  instance_identity: string;
```

Comment:
That's me being picky, but usually it's good to have optional properties after the required ones ;)

Reply:
updated